macros/address.njk

```diff
@@ -111,15 +81,44 @@
              name="{{ enterManuallyName }}"
              />
     </div>
-    <a href="#" class="postcode-enter-manually">{{ t('Enter address manually') }}</a>
+    <a id="{{ name + '[enterManually]' }}" href="#" class="postcode-enter-manually">{{ t('Enter address manually') }}</a>
   </div>
 {% endmacro %}
 
 {% macro addressFragment(form, name) %}
-  {{ textInput(t('Address line 1'), name + '[line1]' if name else 'line1', form, compound=true, autocomplete = 'address-line1') }}
-  {{ textInput(t('Address line 2 (optional)'), name + '[line2]' if name else 'line2', compound=true, form, autocomplete = 'address-line2') }}
-  {{ textInput(t('Town or city'), name + '[city]' if name else 'city', form, compound=true, autocomplete = 'address-level2') }}
-  {{ textInput(t('Postcode'), name + '[postcode]' if name else 'postcode', form, inputClass='postcode', autocomplete = 'postal-code') }}
+  {% set line1Name = name + '[line1]' if name else 'line1' %}
+  {% set line2Name = name + '[line2]' if name else 'line2' %}
+
+  {% set line1Error = form.errorFor(line1Name) %}
+  {% set line2Error = form.errorFor(line2Name) %}
+
+  <div class="form-group{{ ' form-group-error' if line1Error or line2Error else '' }}">
+
+    <label id="{{ line1Name }}[label]" class="form-label" for="{{ line1Name }}">{{ t('Address') }}</label>
```

Comment:
We have `{{ name }}[attribute]` syntax here and `{{ name + '[attribute]'}}` syntax above, would be nice to get it consistent, I don't mind either one.

Reply:
done